Makefile

```diff
@@ -63,3 +63,6 @@ format-mac:
 	cairo-format src/**/*.cairo -i
 	black tests/.
 	isort tests/.
+
+build-sol:
+	docker run -v $(solidity_files):/sources ethereum/solc:stable -o /sources/output --abi --bin --overwrite --opcodes /sources/ERC20.sol /sources/ERC721.sol
```

Comment:
Can we add docker in the prerequisites list in installation README?

Reply:
good point